destreamer.ts

```diff
@@ -264,18 +269,35 @@ async function getVideoInfo(videoID: string, session: any) {
 function getVideoUrls() {
     const t = argv.videoUrls[0] as string;
     const isPath = t.substring(t.length-4) === '.txt';
+    const regex = /(?:https:\/\/)?web.microsoftstream.com\/video\/[a-z0-9]{8}-(?:[a-z0-9]{4}\-){3}[a-z0-9]{12}/;
+    let skip: number[] = [];
     let urls: string[];
 
     if (isPath)
-        urls = fs.readFileSync(t).toString('utf-8').split('\n');
+        urls = fs.readFileSync(t).toString('utf-8').split(/[\r\n]/);

```

Comment:
How about isolating this one change in its own branch so you can send a new PR to fix https://github.com/snobu/destreamer/issues/33?